local_planner/test/test_local_planner.cpp

```diff
@@ -85,9 +88,14 @@ TEST_F(LocalPlannerTests, all_obstacles) {
   // THEN: it should get a scan showing the obstacle
   sensor_msgs::LaserScan scan;
   planner.getObstacleDistanceData(scan);
+  int idx_lower_obstacle_boundary_bin = static_cast<int>(
```

Comment:
This is made possible by using the entire pointcloud for the laser scan, and not limiting it to FOV, right?

Reply:
Actually it would have been possible before:
```
int idx_upper_obstacle_boundary_bin = static_cast<int>(
      (M_PI_F / 2 - atan2(fov_half_y, distance)) / (scan.angle_increment));
```

Reply:
I think you'd have to take a `min()`/`max()` of the FOV and the actual obstacle, depending on the case.